about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css ">
+    <script src="https://kit.fontawesome.com/5116129a4d.js" crossorigin="anonymous"></script>
+    <title>AALA</title>
+</head>
+
+<body>
+    <nav class="navigation">
+                <div class="close">X</div>
+                <ul class="nav-items">
+                    <li class="nav-list"><a href="index.html">Home</a></li>
+                    <li class="nav-list"><a href="about.html">Program</a></li>
+                    <li class="nav-list"><a href="#">Join</a></li>
+                    <li class="nav-list"><a href="#">Sponsor </a></li>
+                    <li class="nav-list"><a href="#">News</a></li>
+                </ul>
+            </nav>
+
+    <section class="sec1" id="sec1">
+          <button class="btn"><i class="fa-solid fa-bars navbtn"></i></button>
+        <div class="desktopdarkheader">
+            <div class="dark-items">
+                <ul>
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><img src="images/twitter.png" alt=""></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </div>
+        <div class="desktoplighterheader">
+            <div class="desktopheader">AALA 2022</div>
+
+            <nav class="references">
+                <a href="about.html" class="about" id="refitem">About</a>
+                <a href="#mainprogramme" class="programme">Program</a>
+                <a href="" class="join">Join</a>
+                <a href="#sec4" class="sponsor">Sponsor</a>
+                <a href="" class="news">News</a>
+            </nav>
+
+        </div>
```

Comment:
Kindly add a link to the home page on the about page navigation menu, since I am unable to go back to the home page from about page 🤗

![image](https://user-images.githubusercontent.com/73714615/190649225-c5a5ddef-11c5-4fda-a3b4-3b612254b15c.png)

Reply:
done